src/components/settings/SettingsAppearanceTab/index.jsx

```diff
@@ -77,10 +159,35 @@ export default function SettingsAppearanceTab({
         <ThemeCard themeVars={ACUMEN_THEME} selected={theme} setTheme={setTheme} />
         <ThemeCard themeVars={PIONEER_THEME} selected={theme} setTheme={setTheme} />
       </CardGrid>
+      <h3 className="py-2 pt-5 mb-0">
+        Custom Theme
+      </h3>
+      {theme[1] === null && (
+      <p className="mt-0">
+        Rather use your own colors?
+        <Button variant="link" onClick={openCustom} className="p-0 pl-1" style={{ verticalAlign: 'top' }}>
+          Create a custom theme.
+        </Button>
+      </p>
+      )}
+      {theme[1] !== null && (
+        <ThemeCard className="mt-1" themeVars={theme[1]} selected={theme} setTheme={setTheme} openCustom={openCustom} />
```

Comment:
nit: just putting out another way to think about the `setTheme` approach here. wouldn't say one is better than other but leaving it as food for thought :) 

Rather than passing `setTheme` to `ThemeCard`, would it be simpler to reason about using callback? E.g., `ThemeCard` accepts an `onSelect` or `onClick` prop. Example:

```jsx
<ThemeCard onSelect={() => setTheme(theme[1])} {/* ... */} />
```

That way, `ThemeCard` doesn't have to know about `setTheme` and makes for a cleaner de-coupling between these components.